presto-hudi/src/main/java/com/facebook/presto/hudi/HudiConfig.java

```diff
@@ -30,6 +31,9 @@
     private boolean sizeBasedSplitWeightsEnabled = true;
     private DataSize standardSplitWeightSize = new DataSize(128, MEGABYTE);
     private double minimumAssignedSplitWeight = 0.05;
+    private int maxOutstandingSplits = 1000;
+    private int splitLoaderParallelism = 2;
+    private int splitGeneratorParallelism = 8;
```

Comment:
These two config entries seem to be confusing. Do we actually need both of them?

Reply:
By default, the user should not fine-tune these two parameters.  We can still keep them in case these need to be changed for performance tuning.